src/parser.rs

```diff
@@ -372,9 +374,14 @@ fn read_atom(token: &str) -> Value {
         return Value::Float(as_float);
     }
 
-    if token.chars().next().map_or(false, |c| c == '"')
-        && token.chars().nth_back(0).map_or(false, |c| c == '"')
-    {
+    if token.chars().next().map_or(false, |c| c == '"') {
+        // Empty string edge-case
+        if token.chars().nth(1).map_or(false, |c| c == '"') {
+            return Value::String(String::from(""));
+        }
```

Comment:
I don't think this edge-case is needed. I added a test for parsing an empty string, and then just deleted this whole clause and it still worked fine

Reply:
Hmm, that's wierd. When I entered an empty string (`""`) in the REPL it came out as (`"""`)

Reply:
Oooooh, if you do `x[a..a]` aka it's a range from a value to the same value, it will give out an empty string as long as we're in a valid range. That's... interesting.